src/frontend/src/utils/urls.ts

```diff
@@ -145,6 +145,9 @@ const designReviewById = (id: string) => `${designReviews()}/${id}`;
 const designReviewDelete = (id: string) => `${designReviewById(id)}/delete`;
 const designReviewMarkUserConfirmed = (id: string) => `${designReviewById(id)}/confirm-schedule`;
 
+/**************** Work Package Template Endpoints ****************/
+const workPackageTemplates = () => `${API_URL}/templates`;
```

Comment:
this doesnt match up with what aaryan used for the template endpoints. The backend endpoint is on the workpackages router which means its off of /work-packages or some shit